frontend/src/components/CytoscapeGraph/ContextMenu/NodeContextMenu.tsx

```diff
@@ -363,15 +363,15 @@ export const clickHandler = (o: ContextMenuOption, kiosk: string) => {
   }
 };
 
-export const getOptions = (node: DecoratedGraphNodeData, jaegerInfo?: JaegerInfo): ContextMenuOption[] => {
+export const getOptions = (node: DecoratedGraphNodeData, jaegerInfo?: TracingInfo): ContextMenuOption[] => {
```

Comment:
should it be `tracingInfo?` or are we indicating here that we are dealing specifically with jaeger information?